Create Dev-Process.md

cjm721 opened the pull request:
Added some basic development process requirements. This will be expanded as needed.

cjm721 committed:
Create Dev-Process.md

Added some basic development process requirements. This will be expanded as needed.

shellajt commented on Dev-Process.md:
```diff
@@ -0,0 +1,79 @@
+* master
+  * f_[Feature_Name]
+    * f_[Feature_name]_[user]
+
```
comment

shellajt commented on Dev-Process.md:
```diff
@@ -0,0 +1,79 @@
+
+### WhiteSpace
+
+There should be no tab character in any of the source code. It should be either 2 or 4 spaces.
```
Do we want to limit ourselves to either 2 or 4? Consistency within our codebase would be nice

xxshadowfang replied:
I think we should specify one standard as well. I like 4 spaces

taylerhow replied:
I agree. I prefer 4 spaces.

shellajt commented:
We need to find a way to lock down master so we cant push to it or merge PRs that havn't been approved

cjm721 committed:
4 Space Vote

cjm721 commented:
Master is already protected so a force push cannot be applied but there are ways to lock it even better.

See [status](https://developer.github.com/v3/repos/statuses/)
And [Integrations](https://github.com/integrations)

shellajt commented:
Lets squash commits before we merge. Might be worth putting into the doc as well

cjm721 commented:
If we squash commits though we lost direct access to the history. So makes looking up specific times very hard. 

If you just don't like the merge commit (which I am not much of a fan of either for small commit groups)  a Fast Forward Merge takes care of that.

I can edit your comment? - Jesse

shellajt commented:
In general, sure. But for something like this were you made an edit to take out two words...

We can take out the tiny commits that arn't important. Small fixes, formatting, etc

Commit:
Add info about agile process

Added a short paragraph about the agile process the team will be using
for the duration of the project.

taylerhow commented:
Looks good!

A few technicalities:
- Friday stand-ups are at 3:30, I believe.
- Are we having Wednesday stand-ups (in addition to meetings w/ Sriram, Sean, retros, and grooming)?

richarel commented:
Looks good to me too. I'll second everything Tayler said regarding the meeting times ^.

Commit:
Added Sirram's feedback

Including corrected meeting times, corrected pull request process, and
corrected review process
